timify-api/server.js

```diff
@@ -0,0 +1,61 @@
+import express from 'express';
+import session from 'express-session';
+import cors from 'cors';
+import morgan from 'morgan';
+import { sequelize } from './database.js';
+import { User} from './models/index.js';
+import userRoutes from './route/users.js';
+import SequelizeStoreInit from 'connect-session-sequelize';
+
+const app = express();
+
+app.use(cors({
+  origin:'http://localhost:5173 ',
+  credentials: true
+}));
+app.use(express.json());
+app.use(morgan())
+
+const SequelizeStore = SequelizeStoreInit(session.Store);
+const sessionStore = new SequelizeStore({
+  db: sequelize
+});
+
+// Session middleware
+app.use(
+  session({
+    secret: 'your-secret-key',
+    resave: false,
+    saveUninitialized: false,
+    store: sessionStore,
+    cookie: {
+      sameSite: false,
+      secure: false,
+      expires: new Date(Date.now() + (365 * 24 * 60 * 60 * 1000)) // 1 year in milliseconds
```

Comment:
Love the comment to explain the numbers! We usually make code more readable by using variables with descriptive names instead of numbers + comments. Would love if you can define a variable at the top of your file that indicates it represents the number of millisecs in a year and use that here instead :)